pkg/agent/cniserver/server.go

```diff
@@ -442,6 +442,10 @@ func (s *CNIServer) CmdAdd(ctx context.Context, request *cnipb.CniCmdRequest) (*
 
 	result := &ipam.IPAMResult{Result: current.Result{CNIVersion: current.ImplementedSpecVersion}}
 	netNS := s.hostNetNsPath(cniConfig.Netns)
+	if !validateRuntime(netNS) {
+		// Support for Docker runtime has been removed since Antrea 2.0.
+		return nil, fmt.Errorf("failed to process CmdAdd request because Docker runtime is not supported after Antrea 2.0")
+	}
```

Comment:
let's clarify that this is for Windows only (in the comment and in the error message)
I imagine that it would still be possible for someone to use https://github.com/Mirantis/cri-dockerd on Linux Nodes if they wanted to